### PR TITLE
[Merged by Bors] - feat: image of any interval under `affineHomeomorph`

### PR DESCRIPTION
**Pull request by scholzhannah**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
Thanks!
maintainer merge

File: Mathlib/Topology/Algebra/Field.lean [hunk 1]
```diff
@@ -91,6 +92,11 @@ def affineHomeomorph (a b : 𝕜) (h : a ≠ 0) : 𝕜 ≃ₜ 𝕜 where
     exact mul_div_cancel_left₀ x h
   right_inv y := by simp [mul_div_cancel₀ _ h]
 
+theorem affineHomeomorph_image_Icc {𝕜 : Type*} [LinearOrderedField 𝕜] [TopologicalSpace 𝕜]
+    [TopologicalRing 𝕜] (a b c d : 𝕜) (h : 0 < a) :
+    affineHomeomorph a b h.ne.symm '' Set.Icc c d = Set.Icc (a * c + b) (a * d + b) := by
```
Comment (jcommelin) [inline Mathlib/Topology/Algebra/Field.lean:97]:
```suggestion
    affineHomeomorph a b h.ne' '' Set.Icc c d = Set.Icc (a * c + b) (a * d + b) := by
```

File: Mathlib/Topology/Algebra/Field.lean [hunk 2]
```diff
@@ -91,6 +92,11 @@ def affineHomeomorph (a b : 𝕜) (h : a ≠ 0) : 𝕜 ≃ₜ 𝕜 where
...
+  simp [h]
+
```
Comment (jcommelin) [inline Mathlib/Topology/Algebra/Field.lean:99]: Could you please add the analogous lemmas for `Set.Ico`, `Set.Ioc`, and `Set.Ioo`?

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**